Cargo.toml

```diff
@@ -12,3 +12,4 @@ rust_decimal_macros = "1.28"
 reqwest = { version = "0.11", features = ["blocking", "json"] }
 serde = { version = "1.0.159", features = ["derive"] }
 serde_json = "1.0.95"
+thiserror = "1.0.40"
```

Comment:
Would this be the preferred version?

Reply:
Thats the latest version and the version I tested the code with.